mtools/mloginfo/sections/connection_section.py

```diff
@@ -84,5 +148,16 @@ def run(self):
             opened = ip_opened[ip] if ip in ip_opened else 0
             closed = ip_closed[ip] if ip in ip_closed else 0
 
-            print "%-15s  opened: %-8i  closed: %-8i" % (ip, opened, closed)
+            if genstats:
+                covered_count = ipwise_count[ip] if ip in ipwise_count else 1
+                connection_duration_ip = ipwise_sum_durations[ip] if ip in ipwise_sum_durations else 0
+
+                print "%-15s  opened: %-8i  closed: %-8i dur-avg(s): %-8i dur-min(s): %-8i dur-max(s): %-8i" % (ip, opened, closed, connection_duration_ip/covered_count, ipwise_min_connection_duration[ip], ipwise_max_connection_duration[ip])
```

Comment:
Would you be able to print them out a stat per line to be consistent with the other outputs of the module. See line 141.

Reply:
this is as per existing format, the stats are printed against their respective ip

Reply:
I got your point added overall stats to stat per line format